content/news/2018-11-14-news.en.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "OpenFisca's latest developments"
+date: "2018-11-14"
+layout: post
+---
+
+Hey! Here's a summary of the latest developments.
+
+<!--more-->
+
+[_[Version française]_](/fr/news/2018-11-14-news)
+
+## International
+
+### Documentation redesign
+
+You can know check in one site OpenFisca plateform guide and Python API official documentation: [doc.openfisca.org](https://doc.openfisca.org) [[Doc #167](https://github.com/openfisca/openfisca-doc/pull/167)]
+
+### Trace, with parameters!
+
+Web API `/trace` endpoint contains law parameters used by every calculated formula so that, to debug a calculation, you won't need to switch between trace result and legislation explorer anymore. [[Core #765](https://github.com/openfisca/openfisca-core/pull/765)]
+
+[Demo: Check new /trace!](https://demo.openfisca.org/legislation/swagger#operations-Calculations-trace)
+
+### Users research
+
+To become better acquainted with OpenFisca users and guide its future evolutions, interviews are conducted. Here is a first [schema](https://drive.google.com/file/d/1-OiVfK0ohxO9qIEJ11gxYKi1_59VJ6oj/view) of the community.  🙂 
+
+
+## France
+
+### Taxes calculation improvements
+
+Improved tax calculation thanks to a comparison between OpenFisca results and state ones produced with simulators like [2018 taxes simulator](https://www3.impots.gouv.fr/simulateur/calcul_impot/2018/complet/index.htm). [[France #1198](https://github.com/openfisca/openfisca-france/pull/1198)]
```

Comment:
Improved -> We improved ?
thanks to a comparison -> by comparing OpenFisca results with the official tax administration simulators, such as…